skaraMirror.sh

```diff
@@ -192,20 +192,15 @@ TAG_SEARCH="jdk-${GITHUB_REPO//[!0-9]/}*+*"
 
 # JDK11+ tag sorting:
 # We use sort and tail to choose the latest tag in case more than one refers the same commit.
-# Versions tags are formatted: jdk-V[.W[.X]]+B; with V, W, X, B being numeric.
-# Transform "-" to "." in tag so we can sort as: "jdk.V[.W[.X]]+B"
-#
-# Example: jdk-11.0.2+2, jdk-11.0.1+3, jdk-11.0.2+1
-# Sort by "B": jdk-11.0.2+1, jdk-11.0.2+2, jdk-11.0.1+3
-# Then, "Stable" sort by V.W.X: jdk-11.0.1+3, jdk-11.0.2+1, jdk-11.0.2+2
-# Latest = jdk-11.0.2+2
-#
+# Versions tags are formatted: jdk-V[.W[.X[.P]]]+B; with V, W, X, P, B being numeric.
+# Transform "-" to "." in tag so we can sort as: "jdk.V[.W[.X[.P]]]+B"
+# Transform "+" to ".0.+" during the sort so that .P (patch) is defaulted to "0" for those
+# that don't have one, and the trailing "." to terminate the 5th field from the +
 # First, sort on build number (B):
-jdk11plus_tag_sort1="sort -t+ -k2n"
-# Second, (stable) sort on (V), (W), (X):
-jdk11plus_tag_sort2="sort -t. -k2n -k3n -k4n -s"
```

Comment:
@andrew-m-leonard is removal of `-s` intentional? Note that `(stable) sort` comment stays.

Reply:
yes, stable sort is implemented on the final sort cmd in a different way using "nl", this is because Solaris platform does not support -s